napi_memory_management.md

```diff
@@ -0,0 +1,131 @@
+# Memory Management
+
+As N-API calls are made, references to objects in the heap for the underlying
+vm may be returned as ```napi_values```.  These references must hold the
```

Comment:
Perhaps we should say "may be returned as variables of type &#96;napi_value&#96;", so that the item in backquotes is exactly the name of the type.